pkgs/stdenv/generic/check-meta.nix

```diff
@@ -40,6 +40,8 @@ let
 
   allowBroken = config.allowBroken or false || builtins.getEnv "NIXPKGS_ALLOW_BROKEN" == "1";
 
+  allowUnsupportedSystem = allowBroken || config.allowUnsupportedSystem or false;
```

Comment:
nitpick: `allowBroken` in my opinion should be kept out of here and be explicitly added in line 182 because brokenness and whether a system is supported are two different things.